tests/compile.js

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+
+// Recompile baseline ink files with the current version of inklecate avilable
+// $PATH.
+
+let childProcess = require('child_process');
+let fs = require('fs-extra');
+let path = require('path');
+
+let fileDirectory = path.join(__dirname, 'files');
+let inkFileDirectory = path.join(fileDirectory, 'original');
+let compiledFileDirectory = path.join(fileDirectory, 'compiled');
+
+// These files require the `-c` flag so that all visits will
+// be counted.
+let filesRequiringCFlag = [
+	'visit_counts_when_choosing.ink',
+	'turns_since_with_variable_target.ink',
+	'read_count_variable_target.ink',
+	'tests.ink'
+]
+
+function getAllInkFiles(dir) {
```

Comment:
Ok for me but if we want less code here, something like [`glob`](https://www.npmjs.com/package/glob) would probably work just as well :)

Reply:
Absolutely. I didn't know if you were okay adding a package like glob. I'll happily rewrite this!

Reply:
As you want, I don't mind it as it is :)

Generally speaking I have 0 issues adding packages as dev dependencies. If they get shipped with inkjs it's a different story :)